AdminUi/src/AdminUi/Controllers/RelationshipsController.cs

```diff
@@ -0,0 +1,49 @@
+﻿using Backbone.AdminUi.Infrastructure.DTOs;
+using Backbone.AdminUi.Infrastructure.Persistence.Database;
+using Backbone.BuildingBlocks.API;
+using Backbone.BuildingBlocks.API.Mvc;
+using Backbone.BuildingBlocks.Application.Abstractions.Exceptions;
+using Backbone.BuildingBlocks.Application.Extensions;
+using Backbone.BuildingBlocks.Application.Pagination;
+using Backbone.Modules.Devices.Application;
+using MediatR;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.Options;
+using ApplicationException = Backbone.BuildingBlocks.Application.Abstractions.Exceptions.ApplicationException;
+
+namespace Backbone.AdminUi.Controllers;
+
+[Route("api/v1/[controller]")]
+[Authorize("ApiKey")]
+public class RelationshipsController : ApiControllerBase
+{
+    private readonly AdminUiDbContext _adminUiDbContext;
+    private readonly ApplicationOptions _options;
+
+
+    public RelationshipsController(IMediator mediator, IOptions<ApplicationOptions> options, AdminUiDbContext adminUiDbContext) : base(mediator)
+    {
+        _adminUiDbContext = adminUiDbContext;
+        _options = options.Value;
+    }
+
+    [HttpGet]
+    [ProducesResponseType(typeof(PagedHttpResponseEnvelope<RelationshipDTO>), StatusCodes.Status200OK)]
+    public async Task<IActionResult> GetAllRelationshipsByParticipantAddress([FromQuery] string participant, [FromQuery] PaginationFilter paginationFilter, CancellationToken cancellationToken)
```

Comment:
I would just call this method `GetAllRelationships`. We may need some additional query parameters in the future, and I wouldn't create a separate route for all of them.